package.json

```diff
@@ -15,9 +15,18 @@
     "webpack": "^5.78.0"
   },
   "devDependencies": {
+    "babel-eslint": "^10.1.0",
     "css-loader": "^6.7.3",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^2.0.0",
     "sass-loader": "^13.2.2",
     "style-loader": "^3.3.2",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
     "webpack-bundle-analyzer": "^4.8.0",
     "webpack-cli": "^5.0.1",
     "webpack-dev-server": "^4.13.2"
```

Comment:
- [ ] While trying to run your project I'm facing following error

<img width="765" alt="Screenshot 2023-04-07 at 11 24 27 AM" src="https://user-images.githubusercontent.com/44798044/230554114-be0ab1ca-4791-41da-aaf3-9ee6e8ad7206.png">

- [ ] Please make sure your  basic commands like `npm run dev`, for building Webpack in development mode, production mode is configured . Webpack dev server is set up and running with no errors. In order to fix this you need to install `html-webpack-plugin` and `html-loader` You can use npm commands `npm install --save-dev html-webpack-plugin` and `npm install --save-dev html-loader` respectively.